File: Main/ReferenceDLLs/Pack1/Facebook Connect Visual Kit VB/FVK Source Code/FVK/FVK/ComponentParts/TabData.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace FVK
    Public Class TabData
        Private m_name As String
        Private m_url As String
        Private m_width As Integer

        Public Sub New(ByVal name As String, ByVal url As String, ByVal width As Integer)
            Me.m_name = name
            Me.m_url = url
            Me.m_width = width
        End Sub

        Public ReadOnly Property Name() As String
            Get
                Return m_name
            End Get
        End Property
        Public ReadOnly Property Url() As String
            Get
                Return m_url
            End Get
        End Property
        Public ReadOnly Property Width() As Integer
            Get
                Return m_width
            End Get
        End Property
    End Class
End Namespace

File: Main/ReferenceDLLs/Pack1/Facebook Connect Visual Kit CS/FVK Source Code/FVK/FVK/ComponentParts/TabData.cs
// WARNING: 
// The source code of this class can be used and modified to your particular needs. 
// Sharing the class with unregistered users or using it with unregistered websites 
// or Facebook applications is not allowed and violates the rights of intellectual 
// property. 

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FVK
{
    public class TabData
    {
        private string name;
        private string url;
        private int width;
       
        public TabData(string name, string url, int width)
        {
            this.name = name;
            this.url = url;
            this.width = width;
        }

        public string Name { get { return name; } }
        public string Url { get { return url; } }
        public int Width { get { return width; } }
    }
}
